README.md

```diff
@@ -1,185 +1,192 @@
 
-### Additional documentation
 
-- [Why use TimescaleDB?](https://tsdb.co/GitHubTimescaleIntro)
-- [Migrating from PostgreSQL](https://docs.timescale.com/migrate/latest/)
-- [Writing data](https://tsdb.co/GitHubTimescaleWriteData)
-- [Querying and data analytics](https://tsdb.co/GitHubTimescaleReadData)
-- [Tutorials and sample data](https://tsdb.co/GitHubTimescaleTutorials)
+# Get involved
 
-### Community & help
+We welcome contributions to TimescaleDB! See [Contributing](https://github.com/timescale/timescaledb/blob/main/CONTRIBUTING.md) and [Code style guide](https://github.com/timescale/timescaledb/blob/main/docs/StyleGuide.md) for details.
 
-- [Slack Channel](https://slack.timescale.com)
-- [Github Issues](https://github.com/timescale/timescaledb/issues)
-- [Timescale Support](https://tsdb.co/GitHubTimescaleSupport): see support options (community & subscription)
+# Learn about Timescale
 
-### Releases & updates
+Timescale is a PostgreSQL database company. To learn more, visit [timescale.com](https://www.timescale.com).
 
- - [Timescale Release Notes](https://tsdb.co/GitHubTimescaleDocsReleaseNotes): see detailed information about current and past
-   versions and subscribe to get
-   notified about new releases, fixes, and early access/beta programs.
+Timescale Cloud is a high-performance, developer focused, cloud platform that provides PostgreSQL services
+for the most demanding AI, time-series, analytics, and event workloads. Timescale Cloud is ideal for production applications and provides high availability, streaming backups, upgrades over time, roles and permissions, and great security.
 
-### Contributing
 
-- [Contributor instructions](https://github.com/timescale/timescaledb/blob/main/CONTRIBUTING.md)
-- [Code style guide](https://github.com/timescale/timescaledb/blob/main/docs/StyleGuide.md)
```

Comment:
Why remove it??

Reply:
These have moved but are still on the page